test/utils/merge-with.test.ts

```diff
@@ -2,20 +2,22 @@ import * as Schema from '../../';
 
 describe('mergeWith()', function () {
   it('should return a merged object schema', () => {
-    const schema = Schema.object({
+    const fooSchema = Schema.object({
       foo: Schema.string()
     }, {
       required: ['foo'],
       additionalProperties: true
     });
 
-    const merged = Schema.mergeWith(schema, Schema.object({
+    const barSchema = Schema.object({
       bar: Schema.string()
     }, {
       required: ['bar'],
       additionalProperties: true,
       nullable: true
-    }));
+    });
+
+    const merged = Schema.mergeWith(fooSchema, barSchema);
```

Comment:
Just a readability change from when I was debugging the mergeWith changes.

Reply:
👍